models/gday/R/model2netcdf.GDAY.R

```diff
@@ -85,32 +85,34 @@ model2netcdf.GDAY <- function(outdir, sitelat, sitelon, start_date, end_date) {
         output[[i]] <- rep(-999, length(t$vals))
     }
     
+    dims <- list(lon = lon, lat = lat, time = t)
+    
     var <- list()
     ## C-Fluxes
-    var[[1]] <- ncvar_def("AutoResp","kgC/m2/s", list(lon,lat,t), -999)
-    var[[2]] <- ncvar_def("HeteroResp", "kgC/m2/s", list(lon,lat,t), -999)
-    var[[3]] <- ncvar_def("TotalResp","kgC/m2/s", list(lon,lat,t), -999)
-    var[[4]] <- ncvar_def("GPP", "kgC/m2/s", list(lon,lat,t), -999)
-    var[[5]] <- ncvar_def("NEE", "kgC/m2/s", list(lon,lat,t), -999)
-    var[[6]] <- ncvar_def("NPP", "kgC/m2/s", list(lon,lat,t), -999)
+    var[[1]] <- PEcAn.utils::to_ncvar("AutoResp",dims)
+    var[[2]] <- PEcAn.utils::to_ncvar("HeteroResp", dims)
+    var[[3]] <- PEcAn.utils::to_ncvar("TotalResp",dims)
+    var[[4]] <- PEcAn.utils::to_ncvar("GPP", dims)
+    var[[5]] <- PEcAn.utils::to_ncvar("NEE", dims)
+    var[[6]] <- PEcAn.utils::to_ncvar("NPP", dims)
     
     ## C-State
-    var[[7]] <- ncvar_def("AbvGrndWood", "kgC/m2", list(lon,lat,t), -999)
-    var[[8]] <- ncvar_def("TotSoilCarb","kgC/m2", list(lon,lat,t), -999)
-    var[[9]] <- ncvar_def("LAI","m2/m2", list(lon,lat,t), -999)
```

Comment:
This is different from the other calls because it has the unit in the second slot. Is that intentional? In general, we should probably use named arguments, rather than relying on positions. But more generally, is it safe to assume that the default unit is "kgC/m2/s"? I would probably be more comfortable with no default argument and keeping the unit explicit.

Reply:
Not sure I follow what you mean by "no default argument and keeping the unit explicit". The units look pretty explicit to me, and those are the output standard's units, so I'm not sure I follow what you think is unsafe?

Reply:
I wasn't sure how the new `PEcAn.utils::to_ncvar` worked. I've since checked, and I see now that it doesn't take a unit argument because it gets those from the standard variables list. So the call `to_ncvar("LAI", "m2/m2", dims)` will not work.

https://github.com/PecanProject/pecan/blob/develop/base/utils/R/to_nc.R#L42

Reply:
Not intentional. Missed deleting this unit.